.gitignore

```diff
@@ -2,3 +2,4 @@ npm-debug.log*
 
 #ignore node_modules, as the node project is not "deployed" per se: http://www.mikealrogers.com/posts/nodemodules-in-git.html
 /node_modules
+.DS_Store
```

Comment:
please ignore this on your own machine with a [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) instead of committing to each repo individually 

```suggestion
```

Reply:
> please ignore this on your own machine with a [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) instead of committing to each repo individually

Really thank you for this new knowledge 🤝